### PR TITLE
feat: use OpenSea for fetching NFTs

### PR DESCRIPTION
**Pull request by Sekhmet**
Closes: https://github.com/snapshot-labs/sx-ui/issues/315

## Screenshots

image

### CODE REVIEW

**Review by bonustrack — COMMENTED** [review]
Nice! We may want to also truncate the NFT name in the selector to get consistent item height
image

File: src/components/NftPreview.vue [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
 const props = defineProps({
   item: Object,
   size: Number
 });
 
-function handleError(e) {
-  const size = props.size ? props.size * 2 : 256;
+const fallbackUrl = computed(
+  () =>
+    `https://cdn.stamp.fyi/token/${props.item.contractAddress}?s=${
```
Comment (bonustrack) [inline src/components/NftPreview.vue:11]: We can use the Stamp NFT endpoint in future to avoid fallback

Comment (Sekhmet) [reply src/components/NftPreview.vue:11]: That PR wasn't merged yet:
https://github.com/snapshot-labs/stamp/pull/28

Comment (bonustrack) [reply src/components/NftPreview.vue:11]: Yep I'll make review there

**Pull request merged**